go/internal/svix_http_client.go

```diff
@@ -26,12 +27,21 @@ type SvixHttpClient struct {
 }
 
 func DefaultSvixHttpClient(defaultBaseUrl string) SvixHttpClient {
+	// Disable HTTP/2.0
+	tr := http.DefaultTransport.(*http.Transport).Clone()
+	tr.ForceAttemptHTTP2 = false
```

Comment:
Based on https://pkg.go.dev/net/http#example-Protocols-Http1or2, it seems like you can just do

`tr.SetHttp2(false)`

Why do we also have to mess with the tls config?